tests/conftest.py

```diff
@@ -8,7 +8,7 @@
 
 try:
     from urllib import parse as urlparse
-except:
+except ImportError:
```

Comment:
Better to use six.moves

Reply:
`six` isn't currently a dependency so, I've left this as is.